### DISCUSSION TITLE
'단락 앞뒤공백'이 적용되지 않습니다.

### DISCUSSION THREAD

**Discussion by sniper0673** [pronist/hELLO#180] [Q&A] [answered]
티스토리에서 '단락앞뒤공백'을 켜도 적용되지 않네요. 
단락앞뒤공백을 적용할 수 있는 방법이 있을까요?

**Comment by pronist** [comment, marked as answer]
@HyuntaMansei 그 옵션은 일관성을 위해 스타일에서 고정 되어있습니다.  `style.css` 에서 `p`  에서 **공백(Margin)**과 관련된 설정을 지워보시기 바랍니다.

**Reply by Cyp9715** [reply]
좋은 스킨 제작해 주셔서 감사합니다. 문제 해결했습니다.
